### PR TITLE
[12.0][MIG] hr_commission

### PR DESCRIPTION
**Pull request by kongrattapong**
done.

### CODE REVIEW

File: hr_commission/models/res_partner.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ class ResPartner(models.Model):
         selection_add=[("salesman", "Salesman (employee)")],
     )
     employee_id = fields.Many2one(
+        string="Employee.",
```
Comment (pedrobaeza) [inline hr_commission/models/res_partner.py:14]: Remove the final dot

File: hr_commission/__manifest__.py [hunk 2]
```diff
@@ -3,10 +3,10 @@
 
 {
     'name': 'HR commissions',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'author': 'Tecnativa,'
               'Odoo Community Association (OCA)',
-    "category": "Commissions",
+    'category': 'Commissions',
```
Comment (pedrobaeza) [inline hr_commission/__manifest__.py]: Better to avoid change quote style

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: hr_commission/models/res_partner.py (hunk 1 above)
Comment (kongrattapong) [reply hr_commission/models/res_partner.py:14]: We got this problem so we need to add dot to make a different
```
2019-07-01 06:18:34,600 13216 WARNING odoo12_hr_commission odoo.addons.base.models.ir_model: Two fields (employee_id, employee) of res.partner() have the same label: Employee. 
2019-07-01 06:18:34,636 13216 WARNING odoo12_hr_commission odoo.addons.base.models.ir_model: Two fields (employee_id, employee) of res.users() have the same label: Employee. 
```

File: hr_commission/__manifest__.py (hunk 2 above)
Comment (kongrattapong) [reply hr_commission/__manifest__.py]: Okay.

File: hr_commission/models/res_partner.py (hunk 1 above)
Comment (pedrobaeza) [reply hr_commission/models/res_partner.py:14]: And where does `employee` come from?

Comment (kongrattapong) [reply hr_commission/models/res_partner.py:14]: employee from res_partner.py in module base.

https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/res_partner.py#L180

Comment (pedrobaeza) [reply hr_commission/models/res_partner.py:14]: OK, that's a new field. We can call this one then `Employee reference`, but I would beg you to check a bit deeper now if there's another link in standard that makes the same function that this `employee_id` field.

Comment (kongrattapong) [reply hr_commission/models/res_partner.py:14]: I checked and found nothing.

Comment (pedrobaeza) [reply hr_commission/models/res_partner.py:14]: OK, thanks, then the other label is better IMO

Comment (kongrattapong) [reply hr_commission/models/res_partner.py:14]: @pedrobaeza thank for your reviews.

**Comment by kongrattapong** [comment]
Does this PR have to change anything more?

**Comment by pedrobaeza** [comment]
You have to comment when changes are done for noticing to reviewers that they can re-review, or it will be unnoticed.

Please squash migration commits into one (https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests), and tell me back.

**Comment by kongrattapong** [comment]
> You have to comment when changes are done for noticing to reviewers that they can re-review, or it will be unnoticed.
> 
> Please squash migration commits into one (https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests), and tell me back.

@pedrobaeza  Oh sorry, I squash already

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As this is very straight-forward, let's merge it:

/ocabot merge

**Pull request merged**